parlai/agents/hugging_face/gpt2.py

```diff
@@ -237,6 +240,12 @@ def add_cmdline_args(
         cls, parser: ParlaiParser, partial_opt: Optional[Opt] = None
     ) -> ParlaiParser:
         agent = parser.add_argument_group("Gpt2 Args")
+        agent.add_argument(
+            "--model_name",
+            type=str,
+            default=None,
```

Comment:
can we provide a default here that would fall back to the existing behavior?

Reply:
Actually, it is already the case. If we don't use the argument "--model_name" at all, it will fall back to the old behavior.